assets/js/blocks/product-categories/block.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { noop } from 'lodash';
```

Comment:
What do you think import this function like that `import noop from 'lodash/noop'` ?

References:
https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark/

Reply:
This is rewritten when built to reference the lodash that's already on the page, as part of WordPress core (using [`@wordpress/dependency-extraction-webpack-plugin`](https://developer.wordpress.org/block-editor/packages/packages-dependency-extraction-webpack-plugin/)). Since `lodash` is already on the page (and not in our bundle), we don't gain anything by importing like that.